### ISSUE TITLE
Add Ed25519 for signing / Include minisgn

### ISSUE THREAD

**Issue by rugk** [theupdateframework/tuf#310]
Is it possible to include [Minisgn](https://jedisct1.github.io/minisign/) (by @jedisct1) into this project or to add Ed25519 signing as a possible option (or even as the default if you want).

Related: Ed25519 was already mentioned in https://github.com/theupdateframework/tuf/issues/43 and [in #119](https://github.com/theupdateframework/tuf/issues/119#issuecomment-71856732)

**Comment by SantiagoTorres** [comment]
Hello, rugk. I think ed25519 is already supported via #289, unless I'm missing additional context.

**Comment by rugk** [comment]
Okay, good to know.

**Comment by vladimir-v-diaz** [comment]
That's correct!  Like Minisgn, we also depend on libsodium (via PyNaCl) to generate Ed25519 keys and signatures.  We also verify Ed25519 signatures in pure Python with PyCA's Ed25519 bindings.

We'll look into Minisgn if we decide to allow external verification of TUF's embedded signatures in the future.

Thanks for the recommendation!
